### ISSUE TITLE
Ranking never updates

### ISSUE THREAD

**Issue by clayrisser** [vdaubry/github-awards#192]
My git ranking is extremely out of date.

Every time I try to update, it never updates.

http://git-awards.com/users/codejamninja

**Comment by Vali0** [comment]
Same here

**Comment by anvaka** [comment]
Yup, looks like it's stale now. @vdaubry any chance you can refresh it?

**Comment by davidjbradshaw** [comment]
This seems like a dead project. I would suggest trying https://gitstar-ranking.com/
